packages/runtime/src/widgets/MultiSelect.tsx

```diff
@@ -77,51 +78,60 @@ const MultiSelect: React.FC<MultiSelectProps> = (props) => {
     {
       setSelectedValues,
       setOptions,
-    },
+    }
   );
-
   return (
-    <SelectComponent
-      allowClear
-      defaultValue={defaultOptions?.map((item) => item.value.toString())}
-      // eslint-disable-next-line react/no-unstable-nested-components
-      dropdownRender={(menu) => <Dropdown menu={menu} newOption={newOption} />}
-      filterOption={(input, option) =>
-        option?.label
-          .toString()
-          .toLowerCase()
-          .startsWith(input.toLowerCase()) || false
-      }
-      mode="tags"
-      notFoundContent={<></>}
-      onChange={handleChange}
-      onSearch={(v) => {
-        if (
-          options.some(
-            (option) =>
-              option?.label
-                ?.toString()
-                .toLowerCase()
-                .startsWith(v.toLowerCase()),
-          )
-        )
-          setNewOption("");
-        else setNewOption(v);
-      }}
-      options={values.options}
-      placeholder={placeholder ?? "Select"}
+    <AntForm.Item
+      label={props?.label}
+      name={props?.id}
       style={{
-        width: styles?.width ?? "100%",
-        margin: styles?.margin,
-        borderRadius: styles?.borderRadius,
-        borderWidth: styles?.borderWidth,
-        borderStyle: styles?.borderStyle,
-        borderColor: styles?.borderColor
-          ? getColor(styles.borderColor)
-          : undefined,
+        margin: "0px",
       }}
-      value={values.value}
-    />
+    >
+      <SelectComponent
+        onChange={handleChange}
```

Comment:
What was the issue with the infinite rendering?

Reply:
I was setting a wrong dependency in onChange which was conflicting with what Form was handling, also the id of the two multi select at different pages ( yaml ) was same causing conflicting default states. Which causes infinite re-redners

Reply:
Got it, yeah the same ID across multiple pages is a known issue #107 I'm planning to address soon